scripts/imgtool/image.py

```diff
@@ -207,34 +246,35 @@ def create(self, key, enckey):
             img = bytes(self.payload[self.header_size:])
             self.payload[self.header_size:] = encryptor.update(img) + \
                                               encryptor.finalize()
+        if protected_tlv_size != 0:
+            self.payload += tlv.get()[protected_tlv_size:]
```

Comment:
I guess it doesn't make a difference to test the value of `protected_tlv_size`, if it is zero you still get `self.payload += tlv.get()[0:]`, the whole list, or no?

Reply:
Yes, you are right. This test is unnecessary.